CONTRIBUTING.md

```diff
@@ -0,0 +1,63 @@
+# Hello Contributors
+
+Thx for your interest! We're so glad you're here.
```

Comment:
We could adopt the preamble from @dcallao's [template PR](https://github.com/equinix-labs/terraform-equinix-template/pull/4/files#diff-eca12c0a30e25b4b46522ebf89465a03ba72a03f540796c979137931d8f92055R1-R6) here

Reply:
I'm going to copy the `Contributing` intro and perhaps that `File Structure` section adapted to this project, but I think it makes sense to keep the `Contributing a new Addon` specific to this project.  The `Contributing via Pull Request` sections could also fit in the @dcallao's template